UnitsNet/CustomCode/UnitClasses/Length.extra.cs

```diff
@@ -37,6 +35,39 @@ public static Length FromFeetInches(double feet, double inches)
         {
             return FromInches((FeetToInches * feet) + inches);
         }
+
+        public static Speed operator/ (Length length, TimeSpan timeSpan)
+        {
+            return Speed.FromMetersPerSecond(length.Meters / timeSpan.TotalSeconds);
+        }
+
```

Comment:
A bit inconsistent code formatting, sometimes empty lines between methods, sometimes not. But don't spend time on this, I'll just do a commit doing a ReSharper code format of the entire solution.